### PR TITLE
Add voltage level buildings peak load

### LINKED ISSUE #955: Add voltage level for electricity building loads
Please add the voltage level (new column: `voltage_level`) according to [our definitions](https://redmine.iks.cs.ovgu.de/oe/projects/ego-n/wiki/Definition_of_thresholds_for_voltage_level_assignment) to the buildings' peak load table `demand.egon_building_electricity_peak_loads`.
Either as postprocessing step or rather in the CTS task?

This had been done in various other tasks already, just search for "voltage_level".

Caveat: @nailend you distinguished by sector which makes it harder to obtain the actual peak load per building (and to assign `voltage_level` to a certain row)... :-/

Any straightforward way? If a full recalculation/redesign of the table is necessary, it might be easier to just sum up the two peak loads (still, I wouldn't know which row the voltage should be assigned to..).

### PR DESCRIPTION
**Pull request by nailend**
Fixes #955 .

## Before merging into `dev`-branch, please make sure that

- [x] the `CHANGELOG.rst` was updated.
- [x] new and adjusted code is formated using `black` and `isort`.
- [x] the `Dataset`-version is updated when existing datasets are adjusted.
- [x] the branch was merged into the
      `continuous-integration/run-everything-over-the-weekend`-branch.
- [x] the workflow is running successful in `test mode`.
- [x] the workflow is running successful in `Everything` mode.

### CODE REVIEW

File: src/egon/data/datasets/electricity_demand_timeseries/cts_buildings.py [hunk 1]
```diff
@@ -247,12 +247,13 @@ class CtsDemandBuildings(Dataset):
...
         super().__init__(
             name="CtsDemandBuildings",
-            version="0.0.0",
+            version="0.0.1",
             dependencies=dependencies,
             tasks=(
                 cts_buildings,
                 {cts_electricity, cts_heat},
                 {get_cts_electricity_peak_load, get_cts_heat_peak_load},
+                assign_voltage_level_to_buildings,
```
Comment (nesnoj) [inline src/egon/data/datasets/electricity_demand_timeseries/cts_buildings.py:256]: I wasn't able to find the task in the current run but it's in the server's codebase :thinking: 
How come?

File: src/egon/data/datasets/electricity_demand_timeseries/cts_buildings.py [hunk 2]
```diff
@@ -1542,10 +1540,70 @@ def get_cts_heat_peak_load():
...
+    ] = 5
+    df_peak_load_buildings.loc[
+        df_peak_load_buildings["peak_load_in_w"] > 5.5 * 1e6, "voltage_level"
+    ] = 4
+    df_peak_load_buildings.loc[
+        df_peak_load_buildings["peak_load_in_w"] > 20 * 1e6, "voltage_level"
+    ] = 3
+    df_peak_load_buildings.loc[
+        df_peak_load_buildings["peak_load_in_w"] > 120 * 1e6, "voltage_level"
+    ] = 1
```
Comment (nesnoj) [inline src/egon/data/datasets/electricity_demand_timeseries/cts_buildings.py:1591]: Actually, you did not include the upper limit in level 7 according to the [intervals as defined](https://redmine.iks.cs.ovgu.de/oe/projects/ego-n/wiki/Definition_of_thresholds_for_voltage_level_assignment). I know, others did it the same way but this shouldn't be an excuse :).
I'll fix this.

**Review by nesnoj — COMMENTED** [review]
I made some comments. As DE run is missing, I'll wait for that..

File: src/egon/data/datasets/electricity_demand_timeseries/cts_buildings.py (hunk 2 above)
Comment (nailend) [reply src/egon/data/datasets/electricity_demand_timeseries/cts_buildings.py]: Thx! Sorry for that laziness. Just copy pasted from the other modules.

Comment (nailend) [reply src/egon/data/datasets/electricity_demand_timeseries/cts_buildings.py]: @nesnoj I don't see any further commits. Is this still open?

File: src/egon/data/datasets/electricity_demand_timeseries/cts_buildings.py (hunk 1 above)
Comment (nailend) [reply src/egon/data/datasets/electricity_demand_timeseries/cts_buildings.py:256]: Git blames myself. Apparently this task and the mapping dissolved by merging them :(

**Comment by nesnoj** [comment]
I removed the CI tick due to the latest fix in 5da3c5532fb94b6604d04ad818c8048a9c24f5e9

**Review by nesnoj — APPROVED** [review]
Works like a charm, thank you @nailend ! :heart:

**Pull request merged**